pkgs/development/python-modules/torch-scatter/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  setuptools,
+  torch,
+}:
+
+buildPythonPackage rec {
+  pname = "torch-scatter";
+  version = "2.1.2";
+  src = fetchFromGitHub {
+    owner = "rusty1s";
+    repo = "pytorch_scatter";
+    tag = version;
+    hash = "sha256-dmJrsWoFsqFlrgfbFHeD5f//qUg0elmksIZG8vXXShc=";
+  };
+  pyproject = true;
+
+  build-system = [ setuptools ];
+
+  dependencies = [ torch ];
+
+  pythonImportsCheck = [ "torch_scatter" ];
+
+  meta = with lib; {
+    description = "PyTorch Extension Library of Optimized Scatter Operations";
+    homepage = "https://github.com/rusty1s/pytorch_scatter";
+    license = licenses.mit;
```

Comment:
add yourself as a maintainer or find a willing adopter

Reply:
done in 110b0aa09b26c012c18e008680ad406151e57a11